scripts/v8-to-v9.py

```
@@ -0,0 +1,178 @@
+    # x/interchainaccounts -> leave unchanged as default state.
+
+    # x/operator -> copy
+    blank_data['app_state']['operator'] = data['app_state']['operator']
+    # drop slash states
+    blank_data['app_state']['operator']['slash_states'] = []
+    # set an empty value for the new field.
+    blank_data['app_state']['operator']['operator_asset_usd_values'] = []
+    # unjail all validators
+    unjailed = []
+    for state in blank_data['app_state']['operator']['opt_states']:
+        if state['opt_info'].get('jailed') == True:
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if state['opt_info'].get('jailed'):` for truth checks

Replace with `state['opt_info'].get('jailed')`

(E712)